go/adbc/pkg/flightsql/driver.go

```diff
@@ -160,12 +160,7 @@ func FlightSQLDatabaseRelease(db *C.struct_AdbcDatabase, err *C.struct_AdbcError
 		return C.ADBC_STATUS_INVALID_STATE
 	}
 	h := (*(*cgo.Handle)(db.private_data))
-
```

Comment:
This is templated, right? Was the template updated?

Reply:
good catch, doing so now